keyboards/haritev2/config.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2024 Dane Lipscombe (@dlip)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 1500U
+
+#define FORCE_NKRO
+
```

Comment:
In general, this should not be set by default, as NKRO is prone to not working in BIOS's.
```suggestion
```